openvpn-wire-protocol.xml

```diff
@@ -1458,12 +1458,13 @@ struct key_exchange {
             <li>bit 4: The client is capable of accepting additional arguments
             to the <xref target="authpending">AUTH_PENDING</xref> message.</li>
             <li>bit 5: The client supports doing feature negotiation in P2P mode.</li>
-            <li>bit 6: The client is capable of parsing and receiving the <xref target="dnsproto">dns</xref> messages pushed option</li>
+            <li>bit 6: The client is capable of parsing and receiving the <xref target="dnsproto">dns</xref> messages pushed option. DEPRECATED. Do not use.</li>
```

Comment:
How is this related to PUSH_UPDATE?  And why does it need to be moved to bit field 11?

Reply:
I agree it would be nice with a note maybe explaining why it's deprecated.

Reply:
It is not related to `push-update` and probably should be moved to another PR. The first implementation of `--dns` didn't properly handle some corner case and we had to add another option.

```
/** Supports the --dns option after all the incompatible changes */
#define IV_PROTO_DNS_OPTION_V2   (1<<11)
```